test/config/FrontendAppConfigSpec.scala

```diff
@@ -67,6 +67,18 @@ class FrontendAppConfigSpec extends SpecBase {
     }
   }
 
+  "deleteNotificationUrl" should {
+    "return correct url for cash account filerole" in new Setup {
```

Comment:
better to use FileRole or fileRole. Same for line 76. Also would be great if we could add tests for DutyDefermentStatement, SecurityStatement and PostponedVATStatement. 

Leave the additional tests if it takes time but should not take much time

Reply:
Updated test description to use FileRole and added tests for other statement types